docs/release_notes/v1.11.1.md

```diff
@@ -0,0 +1,113 @@
+# Dapr 1.11.1
+
+## Fixed an issue with Service Invocation on Kubernetes clusters with the permission enforcement admission controller enabled.
+
+### Problem
+
+Updating Dapr from 1.10 to 1.11 on Kubernetes clusters gives the following error
+when using Service Invocation:
+
+```
+unable to create Dapr service for wrapper, service: test/test-dapr, err:
+services \"test-dapr\" is forbidden: cannot set blockOwnerDeletion if an
+ownerReference refers to a resource you can't set finalizers on:
+```
+
+### Impact
+
+Since v1.11.0 Service Invocation on a subset of Kubernetes clusters is unusable.
+
+### Root cause
+
+When using the permission enforcement admission controller on Kubernetes,
+Service Invocation fails because the operator is unable to create or update
+Services. This is particularly problematic when for OpenShift clusters as they
+have this admission controller enabled by default.
+
+### Solution
+
+Give the operator RBAC permissions to update Deployment and StatefulSet
+finalizers, as well as delete services.
+
+## Fixed a Goroutine leak in Actor reminders and timers.
+
+### Problem
+
+The Daprd sidecar process would increase in memory over time when using Actor
+reminders and timers.
+
+### Impact
+
+Using Actor reminders and timers in v1.11.0, Daprd's memory consumption would
+continuously grow overtime, leading to OOM kills or underutilized nodes.
+
+### Root cause
+
+Goroutines were not being cleaned up when reminders and timers were fired.
+
+### Solution
+
+Goroutines are now cleaned up when reminders and timers are fired.
+
+## MongoDB Actor State Store: Fixed an issue with storing Actor Reminders
+
+### Problem
+
+Actor Reminders without data were stored incorrectly and the empty (null) data was subsequently interpreted as a string value. Everytime an Actor Reminder was updated it reencoded the existing encoding, causing exponential growth in ActorReminder data leading up to the MongoDB document size limit. Other problems include the incorrect storage of the actor reminder period.
+
+### Impact
+
+Since v1.10.5 (and also in v1.11.0), Actor Reminders used with MongoDB as Actor State Store were effectively unusable. While some reminders written or updated with affected versions may work, all reminders written with the impacted versions should be considered unrecoverable as some reminder data will have been lost or corrupted.
```

Comment:
```suggestion
Since v1.10.5 (and also in v1.11.0), Actor Reminders used with MongoDB as Actor State Store were impacted by this bug. While some reminders written or updated with affected versions may work, all reminders written with the impacted versions should be considered unrecoverable as some reminder data may have been corrupted.
```

Reply:
That's already in 1.10 release notes by the way as it was written by me.